rskj-core/src/main/java/co/rsk/core/bc/BlockExecutor.java

```diff
@@ -760,7 +751,7 @@ private BlockResult executeSequential(
                 executedTransactions,
                 receipts,
                 bucketOrder,
-                totalGasUsed,
+                totalGasUsed, // totalBlock = parallel1 + parallel2 + sequential?
```

Comment:
Replace this with new variable `gasUsedInBlock`

Reply:
hmm but this variable is checked in the `init()` method to know if a transaction fits in the block.